appkeeper/datadog_checks/appkeeper/appkeeper.py

```diff
@@ -0,0 +1,108 @@
+import json
+import requests
+from datetime import datetime, timedelta, timezone
+from dateutil.parser import parse
+from datadog_checks.base.errors import CheckException
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+API_URL = 'https://api.appkeeper.sios.com/v2/'
+AUTH_API_URL = API_URL + 'authorize'
+EVENT_API_URL = API_URL + '{}/events'
+INSTANCES_API_URL = API_URL + '{}/instances'
+
+class AppKeeperCheck(AgentCheck):
+    def check(self, instance, now=None):
+        account = instance.get('account')
```

Comment:
The usage of instance argument is discouraged, please use self.instance instead
```suggestion
        account = self.instance.get('account')
```